impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -138,6 +145,9 @@ public void threeTiersCacheManager() throws Exception {
                 )
         ).build(true);
 
+    Cache<Long, String> threeTieredCache = persistentCacheManager.getCache("threeTieredCache", Long.class, String.class);
```

Comment:
I removed 3 System.out calls in this class.  Each System.out was right after the getCache() call.  In addition, the getCache() call was already in Ludovics code so I assumed it was there as an example of how to create a cache(its in the GettingStarted class).